docs/guides/hedgehog-engine/unleashed/levels/glvl.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Sonic GLvl setup tutorial for Sonic Unleashed
+---
+
+# How to use Sonic GLvl with Sonic Unleashed
+
+!!! note
+    This guide covers how to setup Sonic Unleashed levels in GLvl, but does not cover how to use this tool. It's recommended that you also read the [Sonic Generations GLvl guide](/guides/hedgehog-engine/blueblur/levels/glvl.md).
+
+!!! info
+    This guide uses the following tools:
+
+    - [Sonic GLvl](/tools/hedgehog-engine/blueblur/levels/){ target="_blank"}
+    - [Unleashed Mod Toolbox](/tools/hedgehog-engine/unleashed/files){ target="_blank"}  
+    - [Unleashed Object Template](assets/glvl/UnleashedObjectTemplate.zip)
+    - Xbox 360 SDK, find this yourself
+
+#### Preparing .ar Files
+This example will be editing Windmill Isle Act 1 Day. Open your extracted Unleashed game folder and locate your files. In my case the files are 
+````
+#ActD_MykonosAct1.ar.00
+#ActD_MykonosAct1.arl
+ActD_MykonosAct1.ar.00 
+ActD_MykonosAct1.ar.01
+ActD_MykonosAct1.arl
+````
+Once located you will need to copy those files to the #files folder inside your Mod Toolbox directory. now open #SUModToolbox.bat and press 3 and enter to decompress the files, press 1 or 2 depending on the system and press enter.
+ 
+Now copy the files inside #output/DecompressedFiles to a seperate folder. In order for GLvl to read the model you need to make a folder called "Packed" and inside your Unleashed game folder go to "Packed" and copy the level folder to the new "Packed" folder next to your decompressed .ar files.
+
+#### Preparing GLvl
+Locate your Xbox 360 SDK's bin/win32 folder and copy xbcompress.exe, xbdecompress.exe, xbdm.dll, msvcp71.dll and msvcr71.dll and paste them into your GLvls "bin" directory.
+
+Open your GLvl folder and go to `/database/LevelDatabase.xml`, you will need to add your entry for the level, in my case it would be:
+
+`<Entry name="ActD_MykonosAct1" geometry="ActD_MykonosAct1"    layout_merge=""    slot="WindmillIsle 1"     game="Unleashed/">`
+
+Afterwards the file should look something like this:
+````
+<LevelDatabase>
+  <Entry name="ghz200"     geometry="ghz200"        layout_merge=""     slot="GreenHillZone"     game="Generations" />
+  <Entry name="ghz201"     geometry="ghz200"        layout_merge=""     slot="GreenHillZone"     game="Generations" />
+  <Entry name="cte200"     geometry="cte200"        layout_merge=""     slot="CityEscapeZone"    game="Generations" />
+  <Entry name="ActD_Beach" geometry="ActD_Beach"    layout_merge=""     slot="JungleJoyride"     game="Unleashed" />
+  <Entry name="ActD_Africa" geometry="ActD_Africa"    layout_merge=""     slot="SavanaCitadel"     game="Unleashed" />
+  <Entry name="ActD_MykonosAct1" geometry="ActD_MykonosAct1"    layout_merge=""     slot="WindmillIsle 1"     game="Unleashed"/>
+  </LevelDatabase>
```

Comment:
`</LevelDatabase>` has extra white space before it. Remove it in order to have correct identation.